widgets/secrets/src/widget.tsx

```diff
@@ -21,24 +28,21 @@ Stage.defineWidget({
         Stage.GenericConfig.SORT_ASCENDING_CONFIG(true)
     ],
 
-    render(widget, data, error, toolbox) {
+    render(widget, data, _error, toolbox) {
         const { Loading } = Stage.Basic;
 
-        if (_.isEmpty(data)) {
+        if (!data || isEmpty(data)) {
             return <Loading />;
         }
-
-        let formattedData = data;
-        formattedData = {
-            ...formattedData,
-            items: _.map(formattedData.items, item => {
+        const formattedData: FormattedSecrets = {
+            items: _.map(data.items, item => {
                 return {
                     ...item,
                     created_at: Stage.Utils.Time.formatTimestamp(item.created_at),
                     updated_at: Stage.Utils.Time.formatTimestamp(item.updated_at)
                 };
             }),
```

Comment:
We can omit usage of one of the implicit lodash imports 🎉 

```suggestion
            items: data.items.map(item => {
                return {
                    ...item,
                    created_at: Stage.Utils.Time.formatTimestamp(item.created_at),
                    updated_at: Stage.Utils.Time.formatTimestamp(item.updated_at)
                };
            }),
```

Reply:
done ✔️ https://github.com/cloudify-cosmo/cloudify-stage/pull/2357/commits/14c6bc3efade9efb573084247962f81f49cefb5a